shell/platform/darwin/ios/framework/Source/SemanticsObject.mm

```diff
@@ -888,13 +881,12 @@ - (void)dealloc {
 }
 
 - (id)nativeAccessibility {
-  return _platformView;
+  return self.platformView;
 }
 
 @end
 
 @implementation SemanticsObjectContainer {
-  SemanticsObject* _semanticsObject;
```

Comment:
This backing ivar isn't when I swap from `weak` to `assign`.  I included it here because when I migrate to ARC this isn't needed since the `weak` ARC property is auto-synthesized.